ol3-viewer/src/ome/ol3/Viewer.js

```diff
@@ -697,9 +697,11 @@ ome.ol3.Viewer.prototype.addRegions = function(options) {
 	// add a vector layer with the regions
 	if (this.regions_) {
 		this.viewer_.addLayer(new ol.layer.Vector({source : this.regions_}));
-		// enable roi selection by default
+		// enable roi selection by default,
```

Comment:
minor point: could you fix the indentation?
it is a mix of space and tab in this file